pkg/filesystem/fs.go

```diff
@@ -45,6 +46,8 @@ const (
 	DummyMountpoint     string = "/dummy"
 )
 
+var PrefetchMsg globalvar.Message
```

Comment:
This can be replaced by `lru.Cache`. Moreover, we should consider whether global variables should be used.